ui/apps/platform/cypress/integration/vulnerabilities/exceptionManagement/pendingRequestsTable.test.js

```diff
@@ -131,4 +124,178 @@ describe('Exception Management Pending Requests Page', () => {
                     });
             });
     });
+
+    it('should be able to sort on the "Request Name" column', () => {
```

Comment:
The sort testing all are similar. I could make it more reusable. I decided not to do it for now. In a future PR, I'll change it. I think we have a few more sorting-related tests for the other tables.

Reply:
Yeah, that's a good point. Since we decided to not test with the API, a component test might be a better option. Maybe in the future PR we can change that to be a component test.